<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>syslog-ng Config</string>
  </property>
  <widget class="QMenuBar" name="menuBar">
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <addaction name="actionSource"/>
   <addaction name="actionDestination"/>
   <addaction name="actionLog"/>
   <addaction name="actionGlobalOptions"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
    <property name="icon">
      <iconset theme="document-save">
        <normaloff/>
      </iconset>
    </property>
    <property name="text">
      <string>&amp;Save</string>
    </property>
    <property name="shortcut">
      <string>Ctrl+S</string>
    </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionHelp">
    <property name="icon">
      <iconset theme="help">
        <normaloff/>
      </iconset>
    </property>
    <property name="text">
      <string>&amp;Help</string>
    </property>
    <property name="shortcut">
      <string>Ctrl+H</string>
    </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSource">
   <property name="text">
    <string>SRC</string>
   </property>
   <property name="toolTip">
    <string>New source</string>
   </property>
  </action>
  <action name="actionDestination">
   <property name="text">
    <string>DST</string>
   </property>
   <property name="toolTip">
    <string>New destination</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="text">
    <string>LOG</string>
   </property>
   <property name="toolTip">
    <string>New log</string>
   </property>
  </action>
  <action name="actionGlobalOptions">
    <property name="text">
      <string>Global Options</string>
    </property>
    <property name="toolTip">
      <string>Configure global options</string>
    </property>
  </action>
 </widget>
 <resources/>
</ui>
